LoopKitUI/Views/BasalScheduleEntryTableViewCell.swift

```diff
@@ -0,0 +1,336 @@
+//
+//  BasalScheduleEntryTableViewCell.swift
+//  LoopKitUI
+//
+//  Created by Pete Schwamb on 2/23/19.
+//  Copyright © 2019 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+
+protocol BasalScheduleEntryTableViewCellDelegate: class {
+    func basalScheduleEntryTableViewCellDidUpdate(_ cell: BasalScheduleEntryTableViewCell)
+    func validateBasalScheduleEntryTableViewCell(_ cell: BasalScheduleEntryTableViewCell) -> Bool
+}
+
+private enum Component: Int, CaseIterable {
+    case time = 0
+    case whole
+    case fractional
+}
+
+class BasalScheduleEntryTableViewCell: UITableViewCell {
+
+    @IBOutlet weak var picker: UIPickerView!
+
+    @IBOutlet weak var pickerHeightConstraint: NSLayoutConstraint!
+
+    private var pickerExpandedHeight: CGFloat = 0
+
+    @IBOutlet weak var dateLabel: UILabel!
+
+    @IBOutlet weak var unitLabel: UILabel!
+
+    @IBOutlet weak var valueLabel: UILabel!
+
+    public weak var delegate: BasalScheduleEntryTableViewCellDelegate?
+
+    public var minimumBasalRatePerHour: Double = 0
```

Comment:
Might want to have these default values match those in the required initializer for BasalScheduleTableViewController.

Reply:
OBE